tests/OSPSuite.Presentation.Tests/Services/DataImporterSpecs.cs

```diff
@@ -0,0 +1,291 @@
+      }
+
+      private ColumnInfo createConcentrationColumn(ColumnInfo timeColumn)
+      {
+         var concentrationInfo = new ColumnInfo
+         {
+            DefaultDimension = _molarConcentrationDimension,
+            Name = "Concentration",
+            DisplayName = "Concentration",
+            IsMandatory = true,
+            BaseGridName = timeColumn.Name
+         };
+
+         concentrationInfo.SupportedDimensions.Add(_molarConcentrationDimension);
+         concentrationInfo.SupportedDimensions.Add(_massConcentrationDimension);
+         concentrationInfo.SupportedDimensions.Add(_fractionConcentrationDimension);
+         return concentrationInfo;
+      }
+
+      private ColumnInfo createErrorColumn(ColumnInfo timeColumn, ColumnInfo concentrationInfo)
+      {
+         var errorInfo = new ColumnInfo
+         {
+            DefaultDimension = _molarConcentrationDimension,
+            Name = "Error",
+            DisplayName = "Error",
+            IsMandatory = false,
+            BaseGridName = timeColumn.Name,
+            RelatedColumnOf = concentrationInfo.Name
+         };
+
+         errorInfo.SupportedDimensions.Add(_molarConcentrationDimension);
+         errorInfo.SupportedDimensions.Add(_massConcentrationDimension);
+         errorInfo.SupportedDimensions.Add(_dimensionFactory.NoDimension);
+         return errorInfo;
+      }
+   }
+
+   public class When_importing_data_from_correct_configuration : concern_for_DataImporter
+   {
+      protected override void Because()
+      {
+         var parameterList = new List<DataFormatParameter>
+         {
+            new MappingDataFormatParameter("time  [h]",
+               new Column() {Name = "Time", Dimension = _dimensionFactory.Dimension("Time"), Unit = new UnitDescription("h")}),
+            new MappingDataFormatParameter("conc  [mg/l]",
+               new Column()
+               {
+                  Name = "Concentration", Dimension = _massConcentrationDimension, Unit = new UnitDescription("mg/l")
+               }),
+            new MappingDataFormatParameter("SD [mg/l]",
+               new Column()
+               {
+                  Name = "Error", ErrorStdDev = "Arithmetic Standard Deviation", Dimension = _massConcentrationDimension,
+                  Unit = new UnitDescription("mg/l")
+               }),
+            new MetaDataFormatParameter("VenousBlood", "Organ", false),
+            new MetaDataFormatParameter(null, "Molecule", false)
+         };
+         _importerConfiguration.CloneParametersFrom(parameterList);
+      }
+
+      [Observation]
+      public void should_import_simple_data_from_excel()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "IntegrationSample1.xlsx")).Count.ShouldBeEqualTo(1);
+      }
+
+      [Observation]
+      public void use_sheet_names_when_importing_simple_data_from_csv()
```

Comment:
Name does not seem to match what's tested

Reply:
name and test functionality corrected